--- boost_1_52_0/tools/build/v2/user-config.jam	2011-01-25 15:06:12.000000000 -0300
+++ user-config.jam	2012-12-13 16:11:23.000000000 -0300
@@ -44,6 +44,8 @@
 # Configure specific gcc version, giving alternative name to use.
 # using gcc : 3.2 : g++-3.2 ;
 
+# W64-MINGW32 configuration
+using gcc : mingw64 : x86_64-w64-mingw32-g++ ;
 
 # -------------------
 # MSVC configuration.
@@ -90,3 +92,8 @@
 
 # Configure specific Python version.
 # using python : 3.1 : /usr/bin/python3 : /usr/include/python3.1 : /usr/lib ;
+#
+# NO ZLIB and NO BZLIB2 to get rid of windows errors
+modules.poke : NO_BZIP2 : 1 ;
+modules.poke : NO_ZLIB : 1 ;
+modules.poke : NO_COMPRESSION : 1 ;

--- boost_1_52_0/bootstrap.sh	2013-07-22 16:41:32.962159000 -0400
+++ bootstrap.sh	2013-07-22 16:46:47.671264000 -0400
@@ -277,7 +277,7 @@
 
     if test "x$PYTHON_ROOT" = x; then
         echo -n "Detecting Python root... "
-        PYTHON_ROOT=`$PYTHON -c "import sys; print sys.prefix"`
+        PYTHON_ROOT=`$PYTHON -c "import sys; print (sys.prefix)"`
         echo $PYTHON_ROOT
     fi    
 fi

--- boost_1_52_0/libs/iostreams/build/Jamfile.v2	2013-05-26 10:13:58.000000000 -0400
+++ Jamfile.v2	2013-07-22 20:16:02.512088000 -0400
@@ -159,8 +159,6 @@
     : $(sources) 
     : <link>shared:<define>BOOST_IOSTREAMS_DYN_LINK=1 
       <define>BOOST_IOSTREAMS_USE_DEPRECATED
-      [ ac.check-library /zlib//zlib : <library>/zlib//zlib
-        <source>zlib.cpp <source>gzip.cpp ]
     :
     : <link>shared:<define>BOOST_IOSTREAMS_DYN_LINK=1
     ;
